module-common/src/main/resources/db/migration/V4__insert_group_order_table_data.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO table_group (id, created_date) VALUES (1L, '2021-12-15 05:00:00');
+
+INSERT INTO order_table (id, number_of_guests, empty, table_group_id) VALUES (9, 0, true, 1);
+INSERT INTO order_table (id, number_of_guests, empty, table_group_id) VALUES (10, 0, true, 1);
```

Comment:
```suggestion
INSERT INTO order_table (id, number_of_guests, empty, table_group_id) VALUES (10, 0, true, 1);

```

마지막 라인에 개행을 추가하는 것이 POSIX 표준입니다 :)